deploy/helm-preview/services/workflows-service/templates/configmap.yaml

```
@@ -0,0 +1,37 @@
+{{- $postgresqlDatabase := .Values.postgresql.auth.database -}}
+{{- if .Values.workflowService.enabled }}
+apiVersion: v1
+kind: ConfigMap
+metadata:
+    name: {{ .Values.workflowService.nameOverride }}
+    namespace: {{ .Release.Namespace | quote }}
+    labels:
+      app: {{ .Values.workflowService.nameOverride }}
+data:
+  {{- range $key, $value := .Values.workflowService.applicationConfig }}
+  {{- if (eq "BACKOFFICE_CORS_ORIGIN" $key) }}
```

yamllint (1.35.1)

[warning] wrong indentation: expected 0 but found 2

(indentation)